src/main/java/org/folio/circulation/support/http/client/Limit.java

```diff
@@ -0,0 +1,34 @@
+package org.folio.circulation.support.http.client;
+
+import static java.lang.Integer.MAX_VALUE;
+
+import io.vertx.core.buffer.Buffer;
+import io.vertx.ext.web.client.HttpRequest;
+
+public class Limit implements QueryParameter {
```

Comment:
Should it extends `NamedQueryParameter` instead? It seems to me like limit is a sub-class of a named query.

Reply:
`NamedQueryParameter` uses a string for it's value, whereas `PageLimit` uses an integer. 

This could be converted to a string internally. Do you think that compromise is worth it?

Reply:
I'm fine to leave it as is.

Reply:
@bohdan-suprun I think there may well be an argument for further consolidating this, especially as the instances are intended to be immutable.